### DISCUSSION TITLE
connect argocd to clone repo on linux

### DISCUSSION THREAD

**Discussion by Meza1923** [argoproj/argo-cd#8265] [General]
Hey,

There is a way to configure argocd to use clone repository on linux? 

I have cluster with no access to any git source, so I tried to clone my repo to linux vm and use it instead, but I fail.

It will be great if there is any way to accomplish that goal.

thank you a lot.

**Comment by chetan-rns** [comment]
Hi @Meza1923,  you can point Argo CD to a local directory by using the `--local` flag. 

https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_app_sync/#options

**Reply by Meza1923** [reply]
It's seems nice, but it's only when I synced. 
Can I use it as a "repository" or even when I create the app?

Thank you, I'm appreciate your help :)

**Comment by todaywasawesome** [comment]
@Meza1923 Someone did a great talk on air-gapped Argo CD in a community meeting last year. Here's [the link](https://www.youtube.com/watch?v=ixZfewgPZw4).

**Reply by Meza1923** [reply]
Thanks a lot, it was very helpful.
